src/test/java/gov/usgs/owi/nldi/controllers/LinkedDataControllerTest.java

```diff
@@ -140,14 +140,23 @@ public void getFeaturesTest() throws Exception {
 	}
 
 	@Test
-	public void getCharacteristicDataTest() throws IOException {
+	public void getCharacteristicDataWithNullParamsTest() throws IOException {
 		controller.getCharacteristicData(request, response, null, null, null, null);
 		verify(logService).logRequest(any(HttpServletRequest.class));
 		verify(logService).logRequestComplete(any(BigInteger.class), any(int.class));
 		//this is a INTERNAL_SERVER_ERROR because of NPEs that shouldn't happen in real life.
 		assertEquals(HttpStatus.INTERNAL_SERVER_ERROR.value(), response.getStatus());
 	}
 
+	@Test
+	public void getCharacteristicDataWithNonexistingComidTest() throws IOException {
+		controller.getCharacteristicData(request, response, "NowhereSource", "IDontExist", null, null);
+		verify(logService).logRequest(any(HttpServletRequest.class));
+		verify(logService).logRequestComplete(any(BigInteger.class), any(int.class));
+		//this is a INTERNAL_SERVER_ERROR because of NPEs that shouldn't happen in real life.
```

Comment:
This comment doesn't apply to this test.

Reply:
Removed.